app/src/main/java/com/duckduckgo/app/browser/httperrors/HttpErrorPixels.kt

```diff
@@ -72,8 +169,38 @@ class RealHttpErrorPixels @Inject constructor(
         return "${this.pixelName}_count"
     }
 
+    private fun isVpnConnected(
+        netpEntitlementStatus: Boolean,
+        connectionState: ConnectionState,
+        subscriptionStatus: SubscriptionStatus,
+    ): Boolean {
+        if (subscriptionStatus in setOf(
```

Comment:
This is not necessary. We can just check `NetworkProtectionState.isRunning` and it will tell us if our VPN is running when the 5xx occurred.